pages/projects/[id]/index.js

```diff
@@ -112,6 +112,7 @@ const StyledDetailsWrapper = styled.div`
 
 const StyledDescription = styled.p`
   text-align: center;
+  text-align: center;

```

Comment:
```suggestion
  text-align: center;

```
I think once is enough 😊

Reply:
i think so too. 😊